server/utils/configs.js

```diff
@@ -186,7 +186,7 @@ async function testConfigOptions(options) {
           if (info) {
             const { pool: { agent } } = info;
             // find implementation from user agent
-            const implementation = agent.match(/(?<=\/)(\w*)(?=:)/)[0];
+            const implementation = agent.match(/(\w*)(?=:)/)[0];
```

Comment:
This will fail on `Bitcoin ABC`, which has a space in it

Reply:
we're only accounting for the agents we support for now anyway. There are other places where this check will fail for ABC too.

Reply:
`agent.match(/[\w ]*)(?=:)/)[0]` should work here

Reply:
Updated with `([\w\s]*)(?=:)` which seems to catch it correctly.